### PR TITLE
Fct 515 goodstore payment presets

### PR DESCRIPTION
**Pull request by tylermorrisford**
This PR adds Sample Data GoodStore payment draft presets.

### CODE REVIEW

File: models/payment/src/payment/payment-draft/presets/sample-data-goodstore/sebastian-mueller-payment.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    .empty()
+    .customer(
+      KeyReference.presets.customer().key(customerSebastianMueller.key!)
+    )
+    .amountPlanned(Money.random().centAmount(395890).currencyCode('EUR'))
+    .paymentMethodInfo(
+      PaymentMethodInfoDraft.presets
+        .empty()
+        .method('Credit Card')
+        .name(LocalizedString.presets.empty()['de']('Credit Card'))
```
Comment (tylermorrisford) [inline models/payment/src/payment/payment-draft/presets/sample-data-goodstore/sebastian-mueller-payment.ts:31]: I made an assumption that we'd want to use German here - thoughts?

File: models/payment/src/payment/payment-draft/presets/sample-data-goodstore/jennifer-jones-payment.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const jenniferJonesPayment = (): TPaymentDraftBuilder =>
+  PaymentDraft.presets
+    .empty()
+    .customer(KeyReference.presets.customer().key(customerJenniferJones.key!))
+    .amountPlanned(Money.random().centAmount(485759).currencyCode('GBP'))
+    .paymentMethodInfo(
+      PaymentMethodInfoDraft.presets
+        .empty()
+        .method('Debit Card')
+        .name(LocalizedString.presets.empty()['en']('Debit Card'))
```
Comment (stephsprinkle) [inline models/payment/src/payment/payment-draft/presets/sample-data-goodstore/jennifer-jones-payment.ts:29]:
```suggestion
        .name(LocalizedString.presets.empty()['en-US']('Debit Card'))
        .name(LocalizedString.presets.empty()['en-GB']('Debit Card'))
```

File: models/payment/src/payment/payment-draft/presets/sample-data-goodstore/sebastian-mueller-payment.ts (hunk 1 above)
Comment (stephsprinkle) [reply models/payment/src/payment/payment-draft/presets/sample-data-goodstore/sebastian-mueller-payment.ts:31]: I think we should stick with `en-US` and `en-GB` if it's in English.

**Review by stephsprinkle — APPROVED** [review]
Looks good! Just need some localized string adjustments

File: models/payment/src/payment/payment-draft/presets/sample-data-goodstore/jennifer-jones-payment.ts (hunk 2 above)
Comment (tylermorrisford) [reply models/payment/src/payment/payment-draft/presets/sample-data-goodstore/jennifer-jones-payment.ts]: fiiiixed with [ec64776](https://github.com/commercetools/test-data/pull/353/commits/ec64776375f909bdafeecad0341cf119a15ef8da)

**Pull request merged**